honeybee/model.py

```diff
@@ -1349,6 +1351,46 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    def to_hbjson(self, name="unnamed", folder_path=None, indent=None, included_prop=None, triangulate_sub_faces=False):
+        """Writes a Honeybee model to HBJSON.
+
+        Args:
+            name: A text string that will be the name of the HBJSOn. Defaults to "unnamed".
```

Comment:
`n` of `HBJSON` should be uppercase. The same below.

Reply:
Revised